ci/rustfmt.sh

```diff
@@ -13,9 +13,4 @@ VERS=""
 [ "$(which rustup)" != "" ] && VERS="+1.63.0"
 
 # Run fmt
-TMP_FILE=$(mktemp)
-git ls-files | grep '.rs$' | sort >"$TMP_FILE"
-for file in $(comm -23 "$TMP_FILE" rustfmt_excluded_files); do
-	echo "Checking formatting of $file"
-	rustfmt $VERS --edition 2021 --check "$file"
-done
```

Comment:
`CONTRIBUTING.md` needs update to remove reference to `contrib/run-rustfmt.sh`

Reply:
Good one, will fix after rebase